scripts/verify-commits.js

```diff
@@ -0,0 +1,42 @@
+const releaseTypes = ['major', 'minor', 'patch', 'prerelease'];
+let patterns = {
+  major: /<x\.[x|?]\.[x|?]>/,
+  minor: /<\?\.x\.[x|?]>/,
+  patch: /<\?\.\?\.x>/,
+  noRelease: /<[N|n][O|o]>/,
```

Comment:
this could be written as `/<no>/i` :)

Reply:
Great catch! I changed it so patterns can come from package.json as well.